docs/status.md

```diff
@@ -110,7 +110,7 @@ one of the following tracking labels.
 | ReduceWindowOp |no||||no|
 | RemOp |yes||||no|
 | ReplicaIdOp |no||||no|
-| ReshapeOp |no||||no|
+| ReshapeOp |yes||||yes|
```

Comment:
Let's also check whether ReshapeOp's verifier and shape function conform to the newly written spec. If they do, let's flip `yes*` in the table to `yes`. If they don't let's open tickets about this.

Reply:
Done.